test/create_url_fetcher_test_certs.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+cd ${DIR}
+. generate_certs.sh
```

Comment:
Why not `. ${DIR}/generate_certs.sh`? Might be missing a few `${DIR}` below...

I have this horrible feeling that this will break building out of tree, but I'll just fix it up later, I suppose.